flow2ml/Flow.py

```diff
@@ -205,14 +205,22 @@ def getDataset(self,img_dimensions,train_val_split):
 
     return self.dataset
 
-  def deployTensorflowModels(self):
+  def deployTensorflowModels(self,model):
```

Comment:
You are passing only dictionary into this method as a parameter from Readme, but here you are taking model as a parameter. 
I think you should have 2 parameters for this method i.e., model and conversions(dictionary).